### ISSUE TITLE
More than one <main> landmark on error pages

### ISSUE THREAD

**Issue by kasztof** [shopware/platform#19417]
### Shopware Version

6.7.13.0

### Affected area / extension

Platform(Default)

### Actual behaviour

On 404 error page there are two `main` HTML landmarks which is agains web accessibility best practices.

```
<main class="content-main is-error-content"> <--- first main
    <div class="flashbags container"></div>
    <div class="content-main-holder  user-logged-out">
        <div class="container">
            <div class="container-main">
                <main class="content-main"> <--- second main
                    ...
```

The outer `main`, used by all page types is defined in : [src/Storefront/Resources/views/storefront/base.html.twig](https://github.com/shopware/shopware/blob/1798e9b88f221c4b5443c532cd20f39f5eb98538/src/Storefront/Resources/views/storefront/base.html.twig#L65)

For the error pages, there is a second `main` element inside the error templates:
- [src/Storefront/Resources/views/storefront/page/error/error-404.html.twig](https://github.com/shopware/shopware/blob/1798e9b88f221c4b5443c532cd20f39f5eb98538/src/Storefront/Resources/views/storefront/page/error/error-404.html.twig#L4)

### Expected behaviour

Each page should have only one `main` HTML landmark.

### How to reproduce

Enter a 404 URL.
